liblouis/compileTranslationTable.c

```diff
@@ -4940,7 +4945,7 @@ compileTable(const char *tableList, const char *displayTableList,
 			*translationTable = NULL;
 		}
 		if (displayTable) {
-			if (*displayTable) free(*displayTable);
```

Comment:
I always get confused by `*` and `&`. Is the following really functionally equivalent?
``` C
free(*displayTable);
```
versus

``` C
static void freeDisplayTable(DisplayTableHeader *t) {
	free(t);
}
freeDisplayTable(*displayTable);
```
In the first case you are de-referencing the display table and in the second case you aren't. Do I read that correctly?

Reply:
In the first case displayTable is a `DisplayTableHeader **`, in the second case it's a `DisplayTableHeader *`.

Reply:
Oh god have mercy on me